tests/cli/multiple-packages-one-world.wit

```diff
@@ -1,5 +1,5 @@
 // RUN: component embed --dummy --wat % | component wit
-
+package foo:root;
 package test:foo1 {
```

Comment:
This test feels a bit off, now that there's the concept of a "main package" what this test was previously testing isn't quite as applicable any more and it should fail saying the main package has no worlds in it. The test could also be updated to pass a `--world` argument perhaps.